packages/discovery-react-components/src/components/StructuredQuery/cssClasses.ts

```diff
@@ -0,0 +1,6 @@
+import { settings } from 'carbon-components';
+
+export const structuredQueryClass = `${settings.prefix}--structured-query`;
+export const structuredQuerySatisfyRulesDropdownClass = `${settings.prefix}--structured-query__satisfy-rules-dropdown`;
```

Comment:
you could build this off of `structuredQueryClass` -> `${structuredQueryClass}__satisfy-rules-dropdown`